lib/chat/bloc/helper/implementation/ChatMessageProviderHelperImpl.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:farmsmart_flutter/chat/bloc/transformer/implementation/ChatHeaderViewModelTransformer.dart';
+import 'package:farmsmart_flutter/chat/bloc/transformer/implementation/ChatMessageViewModelTransformer.dart';
+import 'package:farmsmart_flutter/chat/model/form/form_entity.dart';
+import 'package:farmsmart_flutter/chat/model/form/form_item_entity.dart';
+import 'package:farmsmart_flutter/chat/ui/widgets/bubble_message.dart';
+import 'package:farmsmart_flutter/chat/ui/widgets/message_circle_avatar.dart';
+import 'package:farmsmart_flutter/chat/ui/widgets/fading_dots.dart';
+import 'package:farmsmart_flutter/chat/ui/widgets/header_message.dart';
+
+import '../ChatMessageProviderHelper.dart';
+
+class _Constants {
+  static const defaultEmptyString = "";
+}
+
+class ChatMessageProviderHelperImpl
+    implements ChatMessageProviderHelper<MessageBubbleViewModel> {
+  @override
+  MessageBubbleViewModel getMessageFromEntity(FormItemEntity entity) {
```

Comment:
It's ok, but remember that you can do this: 
```suggestion
  MessageBubbleViewModel getMessageFromEntity(FormItemEntity entity) => _getMessageAsBubble(entity);
```

Reply:
Maybe we should have some way of make it as a standard. For me is much clear to read with curly brackets {} instead of calling **=>**

Let's decide this and will change to any of 2 options (or maybe some third one 😄 )

Reply:
yeah I have the same issue, the brackets are clearer for me, but seems the standard wants the shorthand.

Reply:
Ok, in this case I will change it to the standard way in case we have small logic as in this case could be a function.